Mathlib/Tactic/Linter/Header.lean

```diff
@@ -76,8 +76,7 @@ It returns the array of all `import` identifiers in `s`. -/
 --   to Mathlib.Init (where this linter is imported)
 -- - that function does not return the Syntax corresponding to each import,
 --   which we use to log more precise warnings.
-partial
-def getImportIds (s : Syntax) : Array Syntax :=
+public partial def getImportIds (s : Syntax) : Array Syntax :=
```

Comment:
(Could you add a comment explaining) why this declaration is public? It looks like this is something internal to the linter.